src/core_plugins/kibana/server/routes/api/management/saved_objects/scroll.js

```diff
@@ -49,7 +49,7 @@ export function registerScrollForExportRoute(server) {
       const savedObjectsClient = req.getSavedObjectsClient();
       const objects = await findAll(savedObjectsClient, {
         perPage: 1000,
-        typesToInclude: req.payload.typesToInclude
+        type: req.payload.typesToInclude
```

Comment:
Yep, this is definitely related, although if I understand this correctly, this issue was present before that PR -- it looks like the original implementation before #19231 was passing `includeTypes` through to the `find` method, rather than `typesToInclude`